website/docs/r/resource_provider.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+subcategory: "Provider"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_resource_provider"
```

Comment:
How about the `azurerm_resource_provider_registration` for this resource name? I suppose this rsource should manages the registration of a resource provider rather than manages a resource provider. What do you think?

Reply:
@ArcturusZhang - Thanks for your feedback. I agree with all your comments, will address it in the next iteration.